### PR TITLE
Update e2e imags, fix gsutil cp command

### PR DESCRIPTION
**Pull request by fejta**
Testing at https://github.com/kubernetes/kubernetes/pull/33463#issuecomment-260057052

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/test-infra/1098)

### CODE REVIEW

File: jobs/ci-fejta-soak-test.sh [hunk 1]
```diff
@@ -19,9 +19,9 @@ set -o pipefail
 set -o xtrace
 
 # Hack to test use a test image
-OVERRIDE="${WORKSPACE}/hack/jenkins/.kubekins_e2e_image_tag"
-mkdir -p $(dirname "${OVERRIDE}")
-echo v20161116-fc014f3 > "${OVERRIDE}"
+#OVERRIDE="${WORKSPACE}/hack/jenkins/.kubekins_e2e_image_tag"
```
Comment (apelisse) [inline jobs/ci-fejta-soak-test.sh]: I have no idea about this.

File: jenkins/e2e-image/e2e-runner.sh [hunk 2]
```diff
@@ -490,6 +490,6 @@ fi
 if [[ "${JENKINS_SOAK_MODE:-}" == "y" && ${E2E_UP:-} == "true" ]]; then
   # We deployed a cluster, save state to gcs
   gsutil cp -a project-private "${HOME}/.kube/config" "${JENKINS_SOAK_PREFIX}/kube-config"
-  gsutil cp - "${JENKINS_SOAK_PREFIX}/release.txt" <(echo "${KUBERNETES_RELEASE}")
-  gsutil cp - "${JENKINS_SOAK_PREFIX}/release-url.txt" <(echo "${KUBERNETES_RELEASE_URL}")
+  echo ${KUBERNETES_RELEASE} | gsutil cp - "${JENKINS_SOAK_PREFIX}/release.txt"
```
Comment (apelisse) [inline jenkins/e2e-image/e2e-runner.sh]: What error did that cause?

File: jobs/ci-fejta-soak-test.sh (hunk 1 above)
Comment (fejta) [inline jobs/ci-fejta-soak-test.sh]: Since I'm updating this image I no longer need to use a different version for these tests

**Comment by fejta** [comment]
Merging per in progress results for https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/33463/pull-kubernetes-e2e-gce/5451

**Pull request merged**